src/d3a/models/strategy/__init__.py

```diff
@@ -202,19 +218,117 @@ def is_eligible_for_balancing_market(self):
                 ConstSettings.BalancingSettings.ENABLE_BALANCING_MARKET:
             return True
 
-    def accept_offer(self, market: Market, offer, *, buyer=None, energy=None,
+    # def sorted_offers_market(self, market_or_id):
```

Comment:
If you want to go the way of communicating between IAAs and markets via redis, you need this, because the Strategies "need" market.sorted_offers. But if the market is disconnected, it will fail.

Reply:
Resolved offline